reporter/osm.py

```diff
@@ -61,26 +64,14 @@ def get_osm_file(bbox, coordinates):
     Equivalent url (http encoded)::
 
     """
-    # This is my preferred way to query overpass since it only fetches back
-    # building features (we would adapt it to something similar for roads)
-    # so it is much more efficient. However it (the retrieved osm xml
-    # file) works for the report but not for the shp2pgsql stuff lower down.
-    # So for now it is commented out. Tim.
-    # url_path = (
-    #     'http://overpass-api.de/api/interpreter?'
-    #     'data=('
-    #     'node["building"]["building"!="no"]'
-    #     '(%(SW_lat)s,%(SW_lng)s,%(NE_lat)s,%(NE_lng)s);'
-    #     'way["building"]["building"!="no"]'
-    #     '(%(SW_lat)s,%(SW_lng)s,%(NE_lat)s,%(NE_lng)s);'
-    #     'relation["building"]["building"!="no"]'
-    #     '(%(SW_lat)s,%(SW_lng)s,%(NE_lat)s,%(NE_lng)s);'
-    #     '<;);out+meta;' % coordinates)
-    # Overpass query to fetch all features in extent
-    url_path = (
-        'http://overpass-api.de/api/interpreter?data='
-        '(node({SW_lat},{SW_lng},{NE_lat},{NE_lng});<;);out+meta;'
-        .format(**coordinates))
+
+    # If no url_path provided, we fetch all osm data in the BBOX
```

Comment:
This comment  / explanation should be in the function docstring for the url_path paramter.